### DISCUSSION TITLE
Welcome jingle on OOBE

### DISCUSSION THREAD

**Discussion by Djani-Dobre** [helloSystem/hello#293] [Ideas]
Hi guys!

Nice work till yet, a Desktop exactly i am looking for. Your vision convinced me.

I am coming to you with an idea, just a simple proposal for version 1 or 2 of your great OS.

I remember when i installed WindowsXP the first time, right after it was initially released, on the very firtst login after setup during the out of the box experience, while the system asked me to create a user, pasword, establishing internet connection and so on, in the background a music was playing. Nothing special but a memorable melody. Every time i set up Windows again i was happy to hear it because it was played only during oobe and i felt like the fresh installed system bid welcome me...

So what is your opinion to integrate something similar in helloSystem? When it is fresh installed and the desktop comes up the first time with the welcome messages in different languages and so on, a memorable song is playing. Onky on this first start.

For example this pleasant and happy song (or a part of it)
https://youtu.be/ustrOyrmLOA

I am sure lot of you knows it and the movie too;)  Its very appropriate because we spent lot of our time with our pcs like they were our friends^^ 
Little joke, but maybe you like the idea or you have an another cool jingle or maybe someone could compose one.

Kind regards

**Comment by probonopd** [comment]
Hello @Djani-Dobre.

Using _that_ melody would be sweet but would probably also land us in legal trouble in no time ;-)

Right now the Welcome Assistant (OOBE in Windows terms) is using
[Pamgaea by Kevin MacLeod](https://incompetech.filmmusic.io/song/4193-pamgaea), https://filmmusic.io/standard-license - (what some would possibly call "[nothing music](https://www.youtube.com/watch?v=AIxY_Y9TGWI)").

So if you know a suitably licensed piece of instrumental music that we could use, please do let us know.

**Reply by Djani-Dobre** [reply]
I see, you are right.

I must missed/overheared this song! Well my initial idea is already integrated and my DELL XPS L502x must have some speaker problems i didnt recognized it. Sorry for that

#_So if you know a suitably licensed piece of instrumental music that we could use, please do let us know._

i will have a look

**Comment by timmrogers** [comment]
How about one of these?

https://freemusicarchive.org/music/Borrtex/Untitled_Love_Story

I like "Being Together".

Or, maybe something from here?

https://freemusicarchive.org/music/Chad_Crouch/Ambient_Atmospheres

Lots and lots of options!

**Comment by Djani-Dobre** [comment]
After some research and deeper thinking about my initiated conversion, my final result is it is solved.
My intention was to explain how great it would be for helloSystem to offer a welcome sound. Well as probonopd mentioned the Welcome sound is already implemented, but due to my laptop i did not hear it.

So, no need for introducing something already exists...i will not start a change request for changing the current song. The team behind this project have 100 other more important things to do and to solve instead of thinking abou changing this current one. By the way, i listended it and in my opinion it is a good choice!

Thanks again for your work and keep it alive!
KR
